test_get_number_of_char (__main__.TestExercice) ... FAIL
test_get_number_of_words (__main__.TestExercice) ... FAIL
test_pair (__main__.TestExercice) ... ok
test_remove_third_char (__main__.TestExercice) ... ok
test_replace_char (__main__.TestExercice) ... ERROR

======================================================================
ERROR: test_replace_char (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 43, in test_replace_char
    output = [exercice.replace_char(v[0], v[1], v[2]) for v in values]
  File "test_exercice.py", line 43, in <listcomp>
    output = [exercice.replace_char(v[0], v[1], v[2]) for v in values]
  File "/home/runner/work/2021a-c01-ch4-exercices-YuliiaOz/2021a-c01-ch4-exercices-YuliiaOz/exercice.py", line 25, in replace_char
    caractere[6] = 'z'
IndexError: list assignment index out of range

======================================================================
FAIL: test_get_number_of_char (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 62, in test_get_number_of_char
    self.assertListEqual(
AssertionError: Lists differ: [-1, -1, -1] != [4, 1, 0]

First differing element 0:
-1
4

- [-1, -1, -1]
+ [4, 1, 0] : Mauvais calcul du nombre d'occurence du caract√®re

======================================================================
FAIL: test_get_number_of_words (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 78, in test_get_number_of_words
    self.assertListEqual(
AssertionError: Lists differ: [0, 0, 0] != [2, 0, 1]

First differing element 0:
0
2

- [0, 0, 0]
+ [2, 0, 1] : Mauvais calcul du nombre d'occurence du mot

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (failures=2, errors=1)
